lib/galaxy/webapps/galaxy/buildapp.py

```diff
@@ -256,6 +256,7 @@ def populate_api_routes( webapp, app ):
     webapp.mapper.resource( 'remote_file', 'remote_files', path_prefix='/api' )
     webapp.mapper.resource( 'group', 'groups', path_prefix='/api' )
     webapp.mapper.resource_with_deleted( 'quota', 'quotas', path_prefix='/api' )
+    webapp.mapper.connect( '/api/interactive_environment_ready', controller='interactive_environment_ready' )
```

Comment:
Eventually we did want to have the ability to run multiple concurrent GIEs. Only one per session right now, though.